src/Commands/Maintenance.php

```diff
@@ -34,7 +34,8 @@ protected function configure()
                 InputArgument::OPTIONAL,
                 'get status or change maintenance mode ( possible values : ' . implode(",", $this->allowed_command_states) . ') ' . PHP_EOL,
                 'status'
-            );
+            )
+            ->addArgument('ipaddress', InputArgument::OPTIONAL, 'ip address to add');
```

Comment:
I'm wondering is using an argument is the good practice or if we should use an option.
I haven't found any clear guidelines on how using them 
Maybe @mickaelandrieu can told us what the good behavior to use ?

Reply:
I've tried an option firt but the syntax is too complex :
php bin/console fop:console:maintenance addip --adressip=127.0.0.1
It's simpliest to do php bin/console fop:console:maintenance addip 127.0.0.1
If you don't put any ip, I use the ask()